src/components/HealthCardDisplay/HealthCardDisplay.js

```diff
@@ -0,0 +1,62 @@
+import React from 'react';
+import { makeStyles } from '@material-ui/core/styles';
+import { Card, CardContent, CardHeader, Grid, Typography } from '@material-ui/core';
+
+const useStyles = makeStyles({
+  bold: {
+    fontWeight: 'bold',
+  }
+});
+
+const HealthCardDisplay = ({ patientData }) => {
+  const classes = useStyles();
+
+  const immunizationCode = codings => {
+    if (codings.length === 0) return '';
+
+    const coding = codings[0];
+    return coding.system ? `${coding.system}#${coding.code}` : coding.code;
+  };
+
+  return (
+    <div>
+      <Typography gutterBottom>
+        <span className={classes.bold}>Name: </span><span>{patientData.name}</span>
+      </Typography>
+      <Typography gutterBottom>
+        <span className={classes.bold}>Date of Birth: </span><span>{patientData.dateOfBirth}</span>
+      </Typography>
+      <Typography variant="h4"> Immunizations </Typography>
+      <Grid
+        container
+        spacing={2}
+        direction="row"
+        justify="flex-start"
```

Comment:
Justify is deprecated - 
![Screen Shot 2021-08-18 at 9 44 23 AM](https://user-images.githubusercontent.com/65190574/129919323-cf8ad77e-d49d-4191-9d87-fb0f8b725701.png)

Reply:
fixed!